scripts/package-python.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/env bash
+
+set -eo pipefail
+
+# Grab the version number
+VERSION=$(cat VERSION)
+
+# Initialize general variables
+PYTHON_DIR="packages/python"
+PROTO_DIR="buda"
+
+# List all proto files
+ENTITIES="$PROTO_DIR/entities/*.proto"
+SERVICES="$PROTO_DIR/services/*.proto"
+
+# Compile entities
+docker run --rm -v $PWD:$PWD -u 1000 -w $PWD \
+    znly/protoc \
+    --python_out=$PYTHON_DIR \
+    -I. $ENTITIES
+
+# Compile services
+docker run --rm -v $PWD:$PWD -u 1000 -w $PWD \
+    bufferapp/grpc-docker-python:latest \
```

Comment:
💯 Thanks for creating the issue, happy to switch to the official image if they support 3.6 :D